quizcon/templates/main/question_form.html

```diff
@@ -1,28 +1,65 @@
 {% extends 'base.html' %}
 {% load bootstrap4 %}
 
-{% block title %}{% if object %}Edit {% else %}New {% endif %} Question{% endblock %}
+{% block title %}Create Question{% endblock %}
 
 {% block pagetitle %}
-    {% if object %}Edit {% else %}New {% endif %} Question
+    Create Question
 {% endblock %}
 
 {% block content %}
-<div id="edit-detail-container" class="row">
-    <div class="col-md-12 order-md-1">
-        <form action="." method="post" v-on:submit="onSubmit">{% csrf_token %}
-
-            {% bootstrap_field form.description %}
-            {% bootstrap_field form.text %}
-            {% bootstrap_field form.explanation %}
-            {% bootstrap_field form.ordinality %}
-            <input type="hidden" name="quiz" value="{{quiz.pk}}" />
-            <div class="row">
-                <div class="col-md-2 mb-1 pr-0">
-                    <button type="submit" class="btn btn-block btn-primary">Save</button>
-                </div>
+<div class="container container-content">
+<!-- Create Question -->
+	<h1>
+		Create Question
+	</h1>
+	<nav aria-label="breadcrumb">
+		<ol class="breadcrumb">
+			<li class="breadcrumb-item"><a href="my-quizzes.html">My Quizzes</a></li>
+			<li class="breadcrumb-item"><a href="edit-quiz.html">{{quiz.title}}</a></li>
+			<li class="breadcrumb-item active" aria-current="page">Question 4</li>
+		</ol>
+	</nav>
+	<form action="." method="post" v-on:submit="onSubmit">{% csrf_token %}
+		<div class="row mb-3">
+			<label for="text" class="col-sm-2 col-form-label">Question</label>
+			<div class="col-sm-10">
+				<textarea class="form-control" id="text" name="text"></textarea>
+			</div>
+		</div>
+		<div class="row mb-3">
+			<label for="thetextarea-a" class="col-sm-2 col-form-label">Answer A</label>
+			<div class="col-sm-10">
+				<textarea class="form-control" id="thetextarea-a"></textarea>
+				<input class="form-check-input" type="radio" name="flexRadioDefault" id="flexRadio-correct-a">
+				<label class="form-check-label" for="flexRadio-correct-a"> Correct </label>
```

Comment:
Oddly, I'm not able to tab to the correct radio buttons... Just something to keep in the back of your head, no changes required right now.